site/static/dashboard.html

```diff
@@ -13,15 +13,13 @@
 <body class="container">
     <div>&gt; <a href="index.html">graphs</a>, <a href="compare.html">compare</a>,
         <a href="dashboard.html">dashboard</a>, <a href="bootstrap.html">bootstrap</a>,
-        <a href="status.html">status</a>, <a href="help.html">help</a>.
+        <a href="status.html">status</a>, <a href="https://github.com/rust-lang/rustc-perf">github</a>,
+        <a href="help.html">help</a>.
     </div>
     <div id="check-average-times"></div>
     <div id="debug-average-times"></div>
     <div id="opt-average-times"></div>
     <div id="as-of"></div>
-    <div style="text-align: center;">
-        <a href="https://github.com/rust-lang-nursery/rustc-perf">Contribute on GitHub</a>
-    </div>
```

Comment:
I think I'd prefer to consolidate to this form, or perhaps just a link from the help page, rather than putting it into the header on each page -- the header seems too important for this kind of link.